Core/src/org/sleuthkit/autopsy/actions/DeleteContentTagAction.java

```diff
@@ -103,4 +118,47 @@ protected void doAction(ActionEvent event) {
     protected void refreshDirectoryTree() {
     }
 
+    protected String getActionDisplayName() {
+        return MENU_TEXT;
+    }
+
+    @NbBundle.Messages({"# {0} - fileID",
+            "DeleteContentTagAction.deleteTag.alert=Unable to untag file {0}."})
+    protected void deleteTag(TagName tagName, ContentTag contentTag, long fileId) {
```

Comment:
Should actionPerformed or something else be calling this method or should it be removed